core/src/main/java/com/scalar/db/api/AbacAdmin.java

```diff
@@ -0,0 +1,786 @@
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Sets the given levels of the given policy to a user that has the given username.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @param levelShortName the short name of the level to set
+   * @param defaultLevelShortName the short name of the default level. If null, the {@code
+   *     levelShortName} will be used as the default level
+   * @param rowLevelShortName the short name of the row level. If null, the {@code
+   *     defaultLevelShortName} will be used as the row level
+   * @throws ExecutionException if the operation fails
+   */
+  default void setLevelsToUser(
+      String policyName,
+      String username,
+      String levelShortName,
+      @Nullable String defaultLevelShortName,
+      @Nullable String rowLevelShortName)
+      throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Adds the given compartment of the given policy to a user that has the given username. Before
+   * adding the compartment, levels must be set to the user.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @param compartmentShortName the short name of the compartment
+   * @param accessMode the access mode
+   * @param defaultCompartment whether the compartment is the default compartment
+   * @param rowCompartment whether the compartment is the row compartment
+   * @throws ExecutionException if the operation fails
+   */
+  default void addCompartmentToUser(
+      String policyName,
+      String username,
+      String compartmentShortName,
+      AccessMode accessMode,
+      boolean defaultCompartment,
+      boolean rowCompartment)
+      throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Removes the given compartment of the given policy from a user that has the given username.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @param compartmentShortName the short name of the compartment
+   * @throws ExecutionException if the operation fails
+   */
+  default void removeCompartmentFromUser(
+      String policyName, String username, String compartmentShortName) throws ExecutionException {
+    throw new UnsupportedOperationException(CoreError.ABAC_NOT_ENABLED.buildMessage());
+  }
+
+  /**
+   * Adds the given group of the given policy to a user that has the given username. Before adding
+   * the group, levels must be set to the user.
+   *
+   * @param policyName the policy name
+   * @param username the username
+   * @param groupShortName the short name of the group
+   * @param accessMode the access mode
+   * @param defaultGroup whether the group is the default group
+   * @param rowGroup whether the group is the row group
```

Comment:
Likewise as my comment on the `addCompartmentToUser` method.
```suggestion
   * @param defaultGroup whether the group is a default group
   * @param rowGroup whether the group is a row group
```